content/en/register/index.adoc

```diff
@@ -5,10 +5,10 @@ draft: false
 images: []
 ---
 
-To start joining membership, you should fill out the https://forms.gle/8iYZoczGARUfUU9A7[form].
+To start joining membership, you should fill out the https://forms.gle/kQNU65HCDqWpjugv5[form].
```

Comment:
Seems like this is incorrect -- just saw the email about the already existing updated form. @gozdesahin could you provide me with the new form you created in this `forms.gle` format?

Reply:
Can be handled in a separate PR.